components/Parsing/Tokenizer.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace PHPDoc\Internal\Parsing;
+
+use PHPDoc\Internal\Parsing\Exception\UnexpectedTokenException;
+
+class Tokenizer implements TokenizerInterface
+{
+    private string $src;
+    private int $nbChars;
+    private int $cursor = 0;
+    private int $line = 1;
+    private array $tokens = [];
```

Comment:
```suggestion
    /** @var array<int, Token> $tokens */
    private array $tokens = [];
```
Peut etre? Ca devrait permettre d'enlever le ``/** @var Token $token */`` devant le foreach

Reply:
Je connaissais pas, merci :)